src/test/java/com/microsoft/sqlserver/jdbc/SQLServerConnectionTest.java

```diff
@@ -434,7 +437,7 @@ public void testClosedConnection() throws SQLException {
         } catch (SQLServerException e) {
             assertEquals(e.getMessage(), TestResource.getResource("R_connectionIsClosed"),
                     TestResource.getResource("R_wrongExceptionMessage"));
-          assertEquals("08S01", e.getSQLState(), TestResource.getResource("R_wrongSqlState"));
+            assertEquals("08S01", e.getSQLState(), TestResource.getResource("R_wrongSqlState"));

```

Comment:
is there a tab here?? (same in line 575)

Reply:
It wasn't formatted properly, it was off by 2 spaces.